src/app/core-components/popup-host/popup-host.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="popup-container-style" [ngClass]="{'noContent-style': isNoContent}">
```

Comment:
Тут просто подсвечу подход с binding классов, на мой взгляд удобный и более читаемый подход
```suggestion
<div class="popup-container-style" [class.noContent-style]="isNoContent">
```